docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -398,25 +426,27 @@ quarkus.oidc.user-info-path=/protocol/openid-connect/userinfo
 quarkus.oidc.introspection-path=/protocol/openid-connect/tokens/introspect
 ----
 
-=== Token Propagation
+[[oidc-token-propagation]]
```

Comment:
I don't see it anywhere, also, can this break existing external links to this section ?

Reply:
@sberyozkin, this section was missing an anchor and so I added anchors similar to how they were applied in other sections. As they are not used, I can remove them again. I'm not sure though what existing external links use instead to point to this section.

Reply:
@sheilamjones Sure, AFAIK, they are only added to support specific links to a given section, otherwise, they are not really necesary I guess